ehb_service/conf/urls.py

```diff
@@ -1,13 +1,13 @@
 from django.conf.urls import *
 from django.contrib import admin
-from django.core.urlresolvers import reverse_lazy
+from django.urls import reverse_lazy
 from django.views.generic.base import RedirectView
 
 admin.autodiscover()
 
-urlpatterns = patterns(
-    '',
-    url(r'^admin/', include(admin.site.urls)),
+urlpatterns = [
+    url(r'^admin/', admin.site.urls),
+    # url(r'^admin/', include(admin.site.urls)),
```

Comment:
Is this a real comment? might need to be removed

Reply:
removed: 70b94dc0577eeb5e2c319a0ef740a793e126e937